integration/configs/integrationTest/messaging_session_test.config.json

```diff
@@ -11,7 +11,7 @@
       "testImpl": "MessagingSessionTest.js",
       "payload": {
         "url": "http://localhost:8080/",
-        "retry": 1,
+        "retry": 2,
```

Comment:
Does this mean it will run three times total or two times total?

Reply:
It runs two times at most.

* If the test passes the first time, it's over.
* If the test fails the first time, it gives it another run.

If retry is set to 1 in config file, it will be set to 5 by default.